### ISSUE TITLE
Libtiff dependancy url not found when installing Wine with brew

### ISSUE THREAD

**Issue by rangarajan-k** [Homebrew/homebrew-core#9402]
Libtiff fails when installing wine because the file is not found in url. Tried all of troubleshooting like cleaning cache, resetting git master, update, cleanup and still same problem.

```
Rangarajans-MBP:~ ranga$ brew doctor
Your system is ready to brew.
Rangarajans-MBP:~ ranga$ brew update
Already up-to-date.
Rangarajans-MBP:~ ranga$ brew install wine
==> Installing dependencies for wine: libtiff, gd, libgphoto2, little-cms2, cmake, jasper, libicns, makedepend, openssl, net-snmp, sane-backends, libtasn1, gmp, nettle, libunistring, libffi, p11-kit, gnutls
==> Installing wine dependency: libtiff
==> Using the sandbox
==> Downloading http://download.osgeo.org/libtiff/tiff-4.0.7.tar.gz
Already downloaded: /Users/ranga/Library/Caches/Homebrew/libtiff-4.0.7.tar.gz
==> Downloading https://mirrors.ocf.berkeley.edu/debian/pool/main/t/tiff/tiff_4.0.7-4.debian.tar.xz

curl: (22) The requested URL returned error: 404 Not Found
Trying a mirror...
==> Downloading https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/t/tiff/tiff_4.0.7-4.debian.tar.xz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "libtiff--patch"
Download failed: https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/t/tiff/tiff_4.0.7-4.debian.tar.xz
Rangarajans-MBP:~ ranga$ 
```

**Comment by jimmypw** [comment]
I've just encountered this same issue. The file has been removed upstream. 

This patch resolves to the new _latest_ version and wine is continuing to install. I'm unsure about the etiquette of submitting a pull request for this sort of thing.

```
diff --git a/Formula/libtiff.rb b/Formula/libtiff.rb
index d6f2aa1..66ef32d 100644
--- a/Formula/libtiff.rb
+++ b/Formula/libtiff.rb
@@ -23,9 +23,9 @@ class Libtiff < Formula
   # Patches from Debian for CVE-2016-10094, and various other issues.
   # All reported upstream, so should be safe to remove this block on next stable.
   patch do
-    url "https://mirrors.ocf.berkeley.edu/debian/pool/main/t/tiff/tiff_4.0.7-4.debian.tar.xz"
-    mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/t/tiff/tiff_4.0.7-4.debian.tar.xz"
-    sha256 "74c9c85b43e1bb1016f96665090da7d8481a48f66a53a43100ab78f729cef0c0"
+    url "https://mirrors.ocf.berkeley.edu/debian/pool/main/t/tiff/tiff_4.0.7-5.debian.tar.xz"
+    mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/t/tiff/tiff_4.0.7-5.debian.tar.xz"
+    sha256 "f4183c48ed74b6c3c3a74ff1f10f0cf972d3dba0f840cf28b5a3f3846ceb2be6"
     apply "patches/01-CVE.patch",
           "patches/02-CVE.patch",
           "patches/03-CVE.patch",

```

**Comment by equal-l2** [comment]
This issue will be solved once #9409 is accepted.

**Comment by ilovezfs** [comment]
#9409 is merged! Thanks for reporting
